### PR TITLE
Map a couple of random methods

### PR DESCRIPTION
**Pull request by SquidDev**
I realise this is incredibly disjoint - sorry. This is just me going through methods that I use and trying to clean them up a little.

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -389,6 +391,7 @@ CLASS bme net/minecraft/block/Block
 		ARG 1 state
 		ARG 2 view
 		ARG 3 pos
+	METHOD d isSolidFullSquare (Lbvc;Lbgl;Lev;Lfa;)Z
```
Comment (liach) [inline mappings/net/minecraft/block/Block.mapping:394]: What is the distinction between small, medium, and full?

Comment (SquidDev) [reply mappings/net/minecraft/block/Block.mapping:394]: Small is used for torches, lanterns and pressure plates - it checks against a `VoxelShape` of size 4x4. Medium is used for rails, repeaters/comparators and pressure plates - 12x12. Full is well, a full block - so 1x1.

I'm aware they're not fantastic names, but I couldn't really find a name which covered all the cases they're used.

File: mappings/net/minecraft/entity/Entity.mapping [hunk 2]
```diff
@@ -305,6 +305,7 @@ CLASS aid net/minecraft/entity/Entity
 	METHOD bm getMaxBreath ()I
 	METHOD bn getBreath ()I
 	METHOD bo getHeadYaw ()F
+	METHOD bp canAttack ()Z
```
Comment (liach) [inline mappings/net/minecraft/entity/Entity.mapping]: `isPlayerAttackable` `canPlayerAttack`

File: mappings/net/minecraft/entity/Entity.mapping [hunk 3]
```diff
@@ -425,6 +426,7 @@ CLASS aid net/minecraft/entity/Entity
 	METHOD r isTeammate (Laid;)Z
 	METHOD s isPartOf (Laid;)Z
 	METHOD t isSpectator ()Z
+	METHOD t onAttacked (Laid;)Z
```
Comment (liach) [inline mappings/net/minecraft/entity/Entity.mapping]: Looks like `handlePlayerAttack(Entity)` to me as this is only used by players and can skip the rest of the player attack logic

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/block/Block.mapping:394]: Consider naming them based on their usages then.

Comment (SquidDev) [reply mappings/net/minecraft/block/Block.mapping:394]: `isSolidForTorchOrLantern` doesn't really roll off the tongue. IMO it's better to describing what it does, and people can look at the implementation if they need to.

Comment (CreepyCre) [reply mappings/net/minecraft/block/Block.mapping:394]: How about something like `isHitboxPole`?

**Comment by modmuss50** [comment]
Updating, any further comments should go in an issue or PR

**Pull request merged**